imblearn/ensemble/easy_ensemble.py

```diff
@@ -4,6 +4,7 @@
 #          Christos Aridas
 # License: MIT
 
+#EasyEnsemble is also called as BalancedBaggingClassifier
```

Comment:
Nop you should make an entry inside the docstring in the "Notes" section and in the "See also"